pack.yaml

```
@@ -3,7 +3,7 @@ name: gitlab
 description: GitLab Rest API
 keywords:
   - gitlab
-version: 1.1.2
+version: 1.1.3
```

Vu qu'on rajoute de nouvelles fonctionnalités, en semver c'est le mineur à incrémenter
```suggestion
version: 1.2.0
```